### ISSUE TITLE
alias_method_chain deprecated in Rails 5

### ISSUE THREAD

**Issue by gdpelican** [ageweke/fortitude#47]
Fortitude appears to be pretty well Rails 5 compatible, with the exception of the use of `alias_method_chain`, which is deprecated and should be replaced with `Module.prepend`. Currently, running fortitude on a Rails 5 app throws a bunch of ugly deprecation warnings on app startup:

```
DEPRECATION WARNING: alias_method_chain is deprecated. Please, use Module#prepend instead. From module, you can access the original method using super. (called from <top (required)> at /Users/<app>/config/environment.rb:5)
```

**Comment by ajb** [comment]
@gdpelican @harlantwood: @vlymar and I are working on a PR for Rails 5 support: https://github.com/ageweke/fortitude/pull/51

We haven't tackled these deprecation errors yet because there were some bigger issues at play,  but we plan to.

If you have a chance, we'd appreciate your thoughts on that PR :)

**Comment by ageweke** [comment]
This has been fixed in 0.9.5, which I just released. It uses `Module#prepend` under the covers on Ruby 2.0 and above, but `alias_method_chain` on Ruby 1.8/1.9.

(Yes, I might be a masochist or insane for supporting Ruby 1.8.7 still…but, honestly, it’s not all that hard, and my experience out in the industry has been that old versions of software stick around for far longer than you’d ever imagine.)
